public/assets/css/style.css

```diff
@@ -159,8 +168,29 @@ h5 {
 }
 
 .cave {
-    background: url(/assets/images/cave_vin.jpg);
+    background: url(/assets/images/cave_vin.jpg) no-repeat fixed;
+    background-size: 100% auto;
+}
+
+.banniere {
+    background: url('/assets/images/vigne-split.jpg') no-repeat;
     background-size: 100% auto;
+    height: 60vh;
+    color: #FDF2E9;
+}
+
+@media screen and (max-width: 1300px){
+    .cave{
+        background: none;
+    }
+
+    .banniere{
+        background: url('/assets/images/vigne.jpg') no-repeat fixed;
+        background-size: 100% auto;
+        height: 50vh;
+        color: #FDF2E9;
+
+    }
```

Comment:
No media queries are allowed, you need to find how to do this with Bootstrap

Reply:
>.<"